core-developers/become-core-developer.rst

```diff
@@ -28,11 +28,12 @@ Gaining commit privileges
 After a candidate has demonstrated consistent contributions, commit privileges
 are granted through these steps:
 
-#. A core developer (submitter, usually the mentor) starts a poll in the
-   `Committers category`_ on the `Python Discourse`_.
+#. A core developer (submitter, usually the mentor) starts a poll
+   (see the :ref:`template <coredev-template>` below) in
+   the `Committers category`_ on the `Python Discourse`_.
 
```

Comment:
I suggest to describe these 4 options:

 - Show who voted: [disabled] (`public=false`)
 - Limit voting to these groups: [committers] (`groups=committers`)
 - Automatically close poll: [in 7 days] (`close=...`)
 - Show results: [When poll is closed] (`results=on_close`)

If we show the code between parenthesis (as I did), maybe the following section ("Poll template") is not needed.

Reply:
The poll template is easiest because you can just copy and paste it. It's a convenience thing. But I'll reword it on your suggestion.